charts/nextcloud/README.md

```diff
@@ -411,3 +416,16 @@ persistence:
   enabled: true
   accessMode: ReadWriteMany
 ```
+
+## Security Contexts
+
+These are all the [SecurityContext](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.27/#securitycontext-v1-core) objects you can configure for this helm chart:
+
+| config option               | Description                                                           |
+|:---------------------------:|:----------------------------------------------------------------------|
+| `nextcloud.securityContext` | Optional SecurityContext for the NextCloud container                  |
+| `cronjob.securityContext`   | Optional SecurityContext for cronjob                                  |
+| `nginx.securityContext`     | Optional SecurityContext for the nginx container in the nextcloud pod |
+
+You can also set the [PodSecurityContext](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.27/#podsecuritycontext-v1-core) for the nextcloud pod.
+By default, `nextcloud.podSecurityContext.fsGroup` is set to `33` (the `www-data` user's GID) by default. Set this to `82` if you're using an alpine nextcloud image.
```

Comment:
Hm, since this is a breaking change it would be nice to place it more prominently.

Reply:
Yeah, we could make a list of breaking changes per version. Of course this should ideally be the changelog file. Maybe a good time to start using one?

Reply:
I think the root is good. I'm not sure how good automatic changelogs are, but we can give it a try.

Reply:
Good enough :)